catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/fragment/ScriptFragmentTest.java

```diff
@@ -238,6 +240,80 @@ public void testCheckControlStructure() {
 				.onCheckBox().check(matches(allOf(not(isChecked()), isEnabled())));
 	}
 
+	@Test
+	public void testCheckControlStructureBySelectingBricks() {
+		openContextualActionModeOverflowMenu();
+		onView(withText(R.string.comment_in_out))
```

Comment:
Maybe clean up this test by calling a function containing a for loop?

Reply:
I refactored the whole file, could you take another look at it? :)

Reply:
I would suggest creating a function in ScriptFragment that passes the brickCount as an argument and call that whenever you want to check if x bricks aren't enabled, you could also provide start and end as parameters if that leads to be more useful in a general way.

Reply:
I extracted the common functionality to the method, could you have a look at m newest changes?